src/worker.py

```
@@ -2952,6 +3205,86 @@ async def api_create_session(req, env):
+    ).bind(session_id, user["id"]).first()
+    if not session:
+        return err("Session not found or access denied", 404)
+
+    try:
+        await env.DB.prepare(
+            "DELETE FROM session_attendance WHERE session_id=?"
+        ).bind(session_id).run()
+        await env.DB.prepare(
+            "DELETE FROM sessions WHERE id=?"
+        ).bind(session_id).run()
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)